### DISCUSSION TITLE
Disabling kid check with jws.Verify on a JWKSet

### DISCUSSION THREAD

**Discussion by rishid** [lestrrat-go/jwx#1083] [General]
I am trying to use `jws.verify` with a KeySet and want to set the kid required to false. The comment mentions adding this `jws.WithKeySetSuboption(jws.WithKeySetRequireKid(false))` as an option but this does not work and it seems like the code doesn't exist. 

Referenced line: https://github.com/lestrrat-go/jwx/blob/ace7a775c6d3fd19723849aa959970d88bd9e5c4/jws/options.go#L139

**Comment by lestrrat** [comment]
The documentation is misleading, but please see the [example here](https://github.com/lestrrat-go/jwx/blob/develop/v2/examples/jws_verify_with_keyset_example_test.go#L42-L45).

**Reply by lestrrat** [reply]
It was probably an overzealous search-and-replace a while back. [It has been fixed.](https://github.com/lestrrat-go/jwx/pull/1090) Thanks for the report
